.github/containers/nox-cross-arch/arm64-ubuntu-20.04-python-3.11.Dockerfile

```diff
@@ -1,3 +1,8 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+# This Dockerfile is used to run the tests in architectures not supported by
+# GitHub Actions.
```

Comment:
The file name says `arm64`, but this comment makes it sound like it can be used for many architectures.

Reply:
* https://github.com/frequenz-floss/frequenz-repo-config-python/pull/167